src/python/bot/minimizer/grammars/JavaScriptBaseLexer.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 Google LLC
```

Comment:
should these files be in bot/tokenizer/grammars ?

Reply:
This is where we have put everything related to antlr grammars. This includes .g4 files as well as the Lexers and Parsers generated and used by antlr. Do you think the files should be split up? Or maybe the directory just needs to be renamed?

Reply:
i mean instead of minimizer directory, use tokenizer root directory. everything inside a grammars sub-dir is fine.

Reply:
I misread that, sorry! I agree, the grammars should be in the same dir as the tokenizer.